### DISCUSSION TITLE
difficulty with a json. it is simple

### DISCUSSION THREAD

**Discussion by Rikmas** [mmalcek/bafi#7] [Q&A]
My json is mpcorto1.json:
{
  "FD-LdPD-112": {
    "lang": "es-AR",
    "S5-Final": "rik"
  }
}

to run bafi.exe
bafi.exe -i mpcorto1.json -t "?{{.FD-LdPD-112.lang}}"

An error occurs
2022/04/07 10:00:24 parseTemplate: template: new:1: bad character U+002D '-'

How can I use tags that have the "-" sign?

Gracias!

Ricardo Mason

**Comment by mmalcek** [comment]
Hi,

I did not know about this issue but it seems to be well known go templates limitation 
here: https://stackoverflow.com/questions/64908171/gomplate-bad-character-u0022
or here: https://stackoverflow.com/questions/63853679/helm-templating-doesnt-let-me-use-dash-in-names

Luckily with simple workaround:

In your case:
You can run it in Windows CMD (not powershell):
```
.\bafi.exe -i mpcorto1.json -t "?{{index . \"FD-LdPD-112\" \"lang\"}}" 
```

Or you can create template file myTemplate.tmpl with content:
```
{{index . "FD-LdPD-112" "lang"}}
```
Then run:
```
.\bafi.exe -i mpcorto1.json -t myTemplate.tmpl
```

I've added this to documentation
https://mmalcek.github.io/bafi/examples/#dashes-in-key-names
